charts/postgres-operator/templates/customrresourcedefinition.yaml

```diff
@@ -36,4 +330,286 @@ spec:
   scope: Namespaced
   subresources:
     status: {}
+  validation:
+    openAPIV3Schema:
+      required:
+        - kind
+        - apiVersion
+        - metadata
+        - configuration
+      properties:
+        kind:
+          enum:
+            - OperatorConfiguration
+        apiVersion:
+          enum:
+            - acid.zalan.do/v1
+        metadata:
+          type: object
+          required:
+            - name
+          properties:
+            annotations:
+              type: object
+              additionalProperties:
+                type: string
+            labels:
+              type: object
+              additionalProperties:
+                type: string
+            name:
+              type: string
+                # Note: the cluster name specified here must begin with the team name.
+            namespace:
+              type: string
+        configuration:
+          type: object
+          properties:
+            etcd_host:
+              type: string
+              nullable: true
+            docker_image:
+              type: string
+            max_instances:
+              type: integer
+              minimum: -1
```

Comment:
imo worth a note `-1` is a special value meaning "no restrictions"

Reply:
added comments